src/main/resources/contract/Activity_RQ_RS.xsd

```diff
@@ -42,6 +43,20 @@
 		</xs:restriction>
 	</xs:simpleType>
 
+	<xsd:element name="ForwardReportToSubscriptionRequest">
+		<xs:complexType>
+			<xs:complexContent>
+				<xs:extension base="ActivityModuleRequest">
+					<xs:sequence>
+						<xs:element name="fishingActivities" type="ram:FishingActivityType" minOccurs="0" maxOccurs="unbounded"/>

```

Comment:
Can you please add the message id(s) to the information sent - we will probably need it later in Subscriptions.